crates/core/src/operations/delete.rs

```diff
@@ -446,7 +446,7 @@ impl std::future::IntoFuture for DeleteBuilder {
                 session.state()
             });
 
-            let state = state_with_file_format_options(state, this.file_format_options.as_ref());
+            let state = state_with_file_format_options(state, this.file_format_options.as_ref())?;
```

Comment:
There's an existing `register_store` function in delta-rs that registers an object store in the DataFusion session. I wonder if we could refactor to put all the DataFusion session config together.

Reply:
That seems like a good idea...